examples/vacanza.py

```
@@ -0,0 +1,323 @@
+    Raises:
+        ValueError: If year range is not valid
+    """
+    try:
+        # year_string in ####-#### format
+        if "-" in year_string:
+            parts = year_string.split("-")
+            start_year = int(parts[0])
+            end_year = int(parts[1])
+
+            if len(parts) != 2:
+                raise ValueError("needs to be START-END format")
```

Ruff (0.15.10)

[warning] Abstract `raise` to an inner function

(TRY301)